lib/session.ts

```diff
@@ -396,6 +396,15 @@ export class Session extends Entity {
         const msg: string = `Unable to create the amqp receiver '${receiver.name}' on amqp ` +
           `session '${this.id}' due to operation timeout.`;
         log.error("[%s] %s", this.connection.id, msg);
+
```

Comment:
There are `reject` cases above too but I'd like to limit the scope of this PR to just timeout for now.